frontend/src/taskTypes.js

```diff
@@ -0,0 +1,8 @@
+export const task_types = [
+    { key: null, value: 'None' },
```

Comment:
```suggestion
    { key: null, value: 'All' },
```
None is misleading, it sounds like no tasks should be shown.